integration-tests/docker/tls/set-docker-host-ip.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCKER_HOST_IP="$(host "$(hostname)" | perl -nle '/has address (.*)/ && print $1')"
+if [ "$DOCKER_HOST_IP" -eq "" ]; then
```

Comment:
I believe `-eq` is for numeric comparisons. To check for an empty string: `[ -z "$DOCKER_HOST_IP" ]`

Reply:
ahhh I thought `shellcheck` would yell at me if I used the wrong comparator type :(